helm_deploy/values-prod.yaml

```diff
@@ -21,25 +21,12 @@ generic-service:
     MANAGE_USERS_API_URL: https://manage-users-api.hmpps.service.justice.gov.uk/
 
   allowlist:
-    analyticplatform-1: 34.250.17.221/32
-    analyticplatform-2: 34.251.212.33/32
-    analyticplatform-3: 34.252.4.39/32
-    ark-dom1-psn1: 51.247.4.0/24
-    ark-dom1-psn2: 51.247.3.0/24
-    dxc_webproxy1: 195.92.38.20/32
-    dxc_webproxy2: 195.92.38.21/32
-    dxc_webproxy3: 195.92.38.22/32
-    dxc_webproxy4: 195.92.38.23/32
-    quantum1: 62.25.109.197/32
-    quantum2: 212.137.36.230/32
-    quantum3: 195.92.38.16/28
-    royal-courts-of-justice: 178.248.34.43/32
-
     groups:
       - prisons
       - probation
       - moj_cloud_platform
       - digital_staff_and_mojo
+      - analytic-platform
```

Comment:
where is the analytic-platform group defined and IP's whitelisted there

Reply:
https://github.com/ministryofjustice/hmpps-ip-allowlists/blob/b88db7075a8379661b9bdc1484f3f2ef15ce737f/ip-allowlist-groups.yaml#L227